src/server/src/main/java/io/cassandrareaper/ReaperApplication.java

```diff
@@ -237,6 +235,22 @@ public void run(ReaperApplicationConfiguration config, Environment environment)
     final NodeStatsResource nodeStatsResource = new NodeStatsResource(context);
     environment.jersey().register(nodeStatsResource);
 
+    HttpClient httpClient = createHttpClient(config, environment);
+    final DiagEventSubscriptionResource eventsResource = new DiagEventSubscriptionResource(context, httpClient);
+    environment.jersey().register(eventsResource);
+    final DiagEventSseResource diagEvents = new DiagEventSseResource(context, httpClient);
+    environment.jersey().register(diagEvents);
+
+    // Enable cross-origin requests for using external GUI applications.
+    if (config.isEnableCrossOrigin() || System.getProperty("enableCrossOrigin") != null) {
+      final FilterRegistration.Dynamic cors
+          = environment.servlets().addFilter("crossOriginRequests", CrossOriginFilter.class);
+      cors.setInitParameter("allowedOrigins", "*");
+      cors.setInitParameter("allowedHeaders", "X-Requested-With,Content-Type,Accept,Origin");
+      cors.setInitParameter("allowedMethods", "OPTIONS,GET,PUT,POST,DELETE,HEAD");
+      cors.addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), true, "/*");
+    }
```

Comment:
i need to move the cors stuff (lines 244-252) back to where it came from (it's not part of the patch)

Reply:
Not sure if that was intentional or not. But I don't mind if you would.